ext/sidecar.h

```diff
@@ -42,6 +43,22 @@ static inline ddog_CharSlice dd_zai_string_to_CharSlice(zai_string str) {
     return (ddog_CharSlice){ .len = str.len, .ptr = str.ptr };
 }
 
+static inline ddog_CharSlice dd_zval_string_to_CharSlice(zval *str) {
+    return (ddog_CharSlice){ .len = Z_STRLEN_P(str), .ptr = Z_STRVAL_P(str) };
+}
+
+static inline ddog_CharSlice dd_str_to_CharSlice(char *str) {
+    return (ddog_CharSlice){ .len = strlen(str), .ptr = str };
+}
+
+static inline zend_string *dd_str_to_zend_string(char *str) {
+    return zend_string_init(ZEND_STRL(str), 0);
+}
+
+static inline zend_string *dd_zval_to_zend_string(zval *str) {
+    return Z_STR_P(str);
+}
+
```

Comment:
```suggestion
static inline ddog_CharSlice dd_str_to_CharSlice(char *str) {
    return (ddog_CharSlice){ .len = strlen(str), .ptr = str };
}

```

Let's cleanup what we don't - and should not - need.